database/ServiceMongoDb.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ilija.savic
+ * Date: 8/23/2017
+ * Time: 3:33 PM
+ */
+
+namespace database;
+
+use exceptions\EmptyTableException;
+use exceptions\InvalidIdException;
+use MongoDB\Driver\Exception\ConnectionTimeoutException;
+use MongoDB\Driver\Query;
+use src\Student;
+
+class ServiceMongoDb implements ServiceInterface
+{
+	private $conn;
+
+	public function __construct($conn)
```

Comment:
typehint what type is $conn (as PHP is not strongly typed, this is not necessary, but highly recommended)